perun-dispatcher/src/test/java/cz/metacentrum/perun/dispatcher/unit/TaskSchedulerTest.java

```diff
@@ -107,11 +81,16 @@ public void simpleRunTest() throws InterruptedException, ExecutionException, Tim
 
 		simpleFutureTask.run();
 		assertFalse(simpleSpy.testFailed);
+		try {
+			// FIXME - make every second test not to fail on interrupted exception
+			Thread.sleep(1000);
```

Comment:
While this seem to work, success or failure of the test is still depending on async behaviour (system it runs on, workload etc.) and it might randomly fail on automatic testing. I would prefer annotating the tests with ``@Ignore`` for now.

Reply:
@ignore put for the whole class.